jcg_testcases/src/main/scala/TestCaseExtractor.scala

```diff
@@ -1,11 +1,68 @@
-import scala.io.Source
-import java.io.PrintWriter
+import org.apache.commons.io.FileUtils
+
 import java.io.File
+import scala.io.Source
+import scala.util.matching.Regex
+
+trait TestCaseExtractor {
+    val pathSeparator: String = File.pathSeparator
+    val debug = false
```

Comment:
Should probably be possible to set this via a command-line flag instead of being hardcoded